lib/internal/fs/streams.js

```diff
@@ -312,7 +311,6 @@ WriteStream.prototype._write = function(data, encoding, cb) {
       }
       return cb(er);
     }
-    this.bytesWritten += bytes;
```

Comment:
notice that WriteStream increments bytesWritten inside `_write()` callback and not in `write()`. I think this is actually wrong and slightly contradicts the previous documentation as well (and possibly expected behaviour, it surprised me).